<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowDyx</class>
 <widget class="QMainWindow" name="MainWindowDyx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionChangeParameter"/>
   <addaction name="separator"/>
   <addaction name="actionStopSimulate"/>
   <addaction name="actionContinueSimulate"/>
   <addaction name="actionEndSimulate"/>
   <addaction name="separator"/>
   <addaction name="action_1Speed"/>
   <addaction name="action_2Speed"/>
   <addaction name="action_4Speed"/>
   <addaction name="action_16Speed"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>返回开始界面</string>
   </property>
  </action>
  <action name="actionChangeParameter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindowDyx/res/dyx_change_parameter.png</normaloff>:/MainWindowDyx/res/dyx_change_parameter.png</iconset>
   </property>
   <property name="text">
    <string>ChangeParameter</string>
   </property>
   <property name="toolTip">
    <string>修改路口参数</string>
   </property>
  </action>
  <action name="actionStopSimulate">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindowDyx/res/dyx_stop_simulate.png</normaloff>:/MainWindowDyx/res/dyx_stop_simulate.png</iconset>
   </property>
   <property name="text">
    <string>StopSimulate</string>
   </property>
   <property name="toolTip">
    <string>停止模拟</string>
   </property>
  </action>
  <action name="actionContinueSimulate">
   <property name="icon">
    <iconset>
     <normalon>:/MainWindowDyx/res/dyx_continue_simulate.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>ContinueSimulate</string>
   </property>
   <property name="toolTip">
    <string>继续模拟</string>
   </property>
  </action>
  <action name="actionEndSimulate">
   <property name="icon">
    <iconset>
     <normalon>:/MainWindowDyx/res/dyx_end_simulate.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>EndSimulate</string>
   </property>
   <property name="toolTip">
    <string>结束模拟</string>
   </property>
  </action>
  <action name="action_1Speed">
   <property name="text">
    <string>×1</string>
   </property>
   <property name="toolTip">
    <string>1倍速</string>
   </property>
  </action>
  <action name="action_2Speed">
   <property name="text">
    <string>×2</string>
   </property>
   <property name="toolTip">
    <string>2倍速</string>
   </property>
  </action>
  <action name="action_4Speed">
   <property name="text">
    <string>×4</string>
   </property>
   <property name="toolTip">
    <string>4倍速</string>
   </property>
  </action>
  <action name="action_16Speed">
   <property name="text">
    <string>×16</string>
   </property>
   <property name="toolTip">
    <string>16倍速</string>
   </property>
  </action>
  <action name="actionChangeTrafficFlow">
   <property name="text">
    <string>调整总车流量</string>
   </property>
   <property name="toolTip">
    <string>模拟车流量大的情况（早晚高峰）或模拟车流量小的情况</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
